src/Templating/version.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>2.2.1</VersionPrefix>
+    <VersionPrefix>2.2.2</VersionPrefix>
```

Comment:
@natemcmaster please confirm it's correct to update this file.

Reply:
Yes, this is right. #4246 tracks removing this redundant file.